### PR TITLE
added feature toggles

### PR DESCRIPTION
**Pull request by ammokhov**
*Issue #, if available:*

*Description of changes:*
added features

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by PatMyron** [comment]
What are these features?

**Review by anshikg — COMMENTED** [review]
What does this property do? I am not clear on the fucntion

**Review by anshikg — APPROVED** [review]

**Review by wbingli — APPROVED** [review]

File: src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
     private Boolean snapshotRequested;
     private Boolean rollback;
     private Boolean driftable;
+    private Map<String, Object> features;
```
Comment (PatMyron) [inline src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java:49]: Curious what the `Object` might look like

Comment (ammokhov) [reply src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java:49]: ideally, any primitive/nested map

**Pull request merged**